package-lock.json

```diff
@@ -1,13682 +1,8 @@
 {
   "name": "c2c_ui",
   "version": "7.6.10",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
please use recent node version and keep lockfile v2. Current node LTS is 16 and this is the version targeted in CI.